src/builder/mod.rs

```diff
@@ -122,7 +122,7 @@ impl Builder {
     pub(crate) fn generate(
         &mut self,
         target: generator::Target,
-        out_file: PathBuf,
+        buffer: &mut std::boxed::Box<impl Write>,
```

Comment:
Why you can't just use `impl Write`?

Reply:
Indeed, it compiles fine without it

```suggestion
        buffer: &mut impl Write,
```